src/KeyUnit.cpp

```diff
@@ -60,12 +60,14 @@ void KeyUnit::uninstall(const QString& packageName)
 
 bool KeyUnit::processDataStream(int key, int modifier)
 {
+    bool isAtLeastOneMatch = false;
```

Comment:
Sorry - but changing the core behaviour of how keybindings work is not in scope of https://github.com/Mudlet/Mudlet/issues/1068 at all nor do I want to have a discussion on changing the behaviour on the last day PRs can go into a release. Could you move this to a separate PR for the next release?